itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestJdbcDriver2.java

```diff
@@ -3422,4 +3422,26 @@ public void testHeaderFooterNonTextFiles() throws Exception {
       stmt.close();
     }
   }
+  /**
```

Comment:
add all the use cases like partitioned , non-partitioned and load data use cases.